src/OrchardCore.Modules/Orchard.OpenId/README.md

```diff
@@ -103,3 +103,25 @@ OpenID Connect apps require the following configuration.
       "AllowHybridFlow": false
 }
 ```
+
+## Configuring Certificates
+
+### Windows / IIS
+
+Several tools are availble for generating a signing certificate on Windows and/or IIS, for example:
+
++ IIS Server Manager _(offers not much control)_
+    1. Server Certificates
+    2. Create Self-Signed Certificate 
++ PowerShell _(offers full control)_
+    1. `New-SelfSignedCertificate` https://technet.microsoft.com/en-us/itpro/powershell/windows/pkiclient/new-selfsignedcertificate
```

Comment:
Hum, maybe we should remove `-KeyExportPolicy NonExportable` for a "public"/general example. Folks who decide to deploy their Orchard app to multiple servers might have a bad surprise :sweat_smile: 

(that said, even `NonExportable` certificates *can* be exported, using tools like `mimikatz` or https://github.com/iSECPartners/jailbreak, it's just a bit more complicated :sweat_smile:)